math/binary/abs.go

```diff
@@ -0,0 +1,7 @@
+package binary
+
+// ABS - this function return absolute value use binary operation
+func ABS(base, n int) int {
+	m := n >> (base - 1)
```

Comment:
Explain these bit-wise operations in comments.

Reply:
Done